synapse/lib/ast.py

```diff
@@ -4793,9 +4783,9 @@ async def run(self, runt, genr):
             try:
                 oldv = node.get(name)
                 valu = await rval.compute(runt, path)
-                newv, norminfo = prop.type.normVirt(virts[0], oldv, valu)
+                newv, opts = prop.type.normVirt(virts[0], oldv, valu)
 
-                await node.set(name, newv, norminfo=norminfo)
+                await node.set(name, newv, **opts)
```

Comment:
Any concern that this preclude us from adding things to the `opts` that are not valid kwargs to `Node.set()`? Seems like it also breaks the "norm functions return (norm, norminfo) and norminfo gets handed into set" pattern that we use elsewhere?

Reply:
moved 'merge' into norminfo